src/js/pages/VoterGuide/OrganizationVoterGuide.jsx

```diff
@@ -136,6 +139,30 @@ export default class OrganizationVoterGuide extends Component {
   }
 
   componentDidUpdate (prevProps) {
+    const { dataLayerFired } = this.state;
+    if (!dataLayerFired && VoterStore.voterFirstRetrieveCompleted()) {
+      const { location: { pathname } } = this.props;
+      const currentPage = getPageDetails();
+      
+      TagManager.dataLayer({
+        dataLayer: {
+          event: 'landing',
```

Comment:
Please add actionDetails in your next pull request. Please update in your next pull request to follow new pageDetails and userDetails structure (search the code base to see how others are doing it now.)